pages/githubandmarkdown.md

```diff
@@ -60,6 +60,8 @@ After you rename it, make sure to post a link to the github.io website you've ju
 
 ![Example of github.io page](uploads/images/examplegithubio.png)
 
+Don't worry if you see a **404** when you access the link, you will set that up in *Step 5*.
```

Comment:
404 didn't come from step 5, but it is due to Github needs time to set up our Github page.

CMIIW 😄